src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrOfProps = sourceString.split(';')
```

Comment:
тип змінної у назві краще не писати
```suggestion
  const props = sourceString.split(';')
```